pretests/docker_test_images/docker_test_images.py

```diff
@@ -0,0 +1,80 @@
+r"""
+Summary
+---------
+
+Pre-testing pulling and listing of test image(s) into sysinfo results
+
+Operational Summary
+----------------------
+
+#. Parse the default test image into FQIN format
+#. Pull the default, and any configured ``extra_fqins_csv`` images
```

Comment:
So if I understand correctly, this introduces a new `extra_fqins_csv`. Why not just use the existing `preserve_fqins`, and not do error checks on pulls?